sirepo/job_supervisor.py

```diff
@@ -257,9 +254,44 @@ async def receive(cls, req):
                 return cls._reply_exception(e)
             raise
 
-    async def op_run_timeout(self, op):
+    async def op_send_timeout(self, op):
         pass
 
+    async def _cancel_op(self, to_cancel):
+        def _create_op():
+            # _create_op does too much and expects a request
+            return _Op(
+                _supervisor=_Supervisor(),
+                api="cancel_or_timeout",
+                driver=to_cancel.driver,
+                is_destroyed=False,
+                max_run_secs=None,
+                msg=PKDict(opIdsToCancel=[to_cancel.op_id]),
+                op_name=job.OP_CANCEL,
+                uid=to_cancel.driver.uid,
+            )
+
```

Comment:
When I set the SIREPO_JOB_SUPERVISOR_MAX_SECS_IO to a small value, and try to download a file, it fails with a "413 content to large". Is that the expected error, or should it be some kind of http timeout?

Reply:
Thanks for testing. 

698e3a59ef34eb3b73284cff515b89d3be2f0f47 added this to job_api:

```py
                if r.state == sirepo.job.CANCELED:
                    # POSIT: Users can't cancel donwloadDataFile. So canceled means there was a
                    # timeout (max_run_secs exceeded).
                    raise _content_too_large(req)
```

Reply:
I'll open a new issue for this - it would be too confusing on the client if it returns "content too large" for a timeout. Even a "resource unavailable" or similar would be less confusing.